apps/service_providers/views.py

```diff
@@ -230,11 +147,15 @@ def _get_models_by_type(queryset):
 @login_and_team_required
 @permission_required("service_providers.add_llmprovidermodel")
 def create_llm_provider_model(request, team_slug: str):
-    form = LlmProviderModelForm2(request.POST)
+    form = LlmProviderModelForm(request.team, request.POST)
     if form.is_valid():
         model = form.save(commit=False)
         model.team = request.team
         model.save()
+    else:
+        if len(form.errors) == 1:
```

Comment:
should this be `> 1`?

Reply:
No, this is a dict of 'field' -> list[error]. But I do only want to do this if the single field is `__all__` otherwise you won't know which field the error relates to